content/en/docs/how-to/registry-authentication.md

```diff
@@ -7,74 +7,54 @@ weight: 1
 
 You can use `notation` to authenticate to an OCI-compliant registry. This is useful for pushing and pulling signed artifacts, as well as inspecting artifacts in registries that do not offer public access.
```

Comment:
Question: Is Notation useful for pushing and pulling signed artifacts, or notation authentication? 
Question: Need more elaboration on what does this phrase mean - "...inspecting artifacts in registries that do not offer public access".  This last phrase  makes it sounds like a side door, which isn't the case.

Reply:
> * `notation` can push/pull signed artifacts _AND_ do auth. I believe the push/pull uses ORAS to accomplish this

I agree with @iamsamirzon that notation doesn't push/pull signed artifacts, at least this is current behavior.  @zr-msft you can take a look at this picture https://github.com/notaryproject/notaryproject/blob/55176e81602703451f720b231496e8c346100d46/media/notation-sign-remote.svg

> no public access is intended to mean a web-frontend for viewing artifacts.

`Public access` here causes confusion. I would suggest we rephrase it or just remove `that do not offer public access.`

Reply:
thank you @yizha1  for confirming and @iamsamirzon  for the feedback.

Fixed